### PR TITLE
updated docs for object management

### PR DESCRIPTION
**Pull request by haripriya9647**
#44501

### CODE REVIEW

File: content/en/docs/concepts/overview/working-with-objects/object-management.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ Disadvantages compared to imperative commands:
 Advantages compared to declarative object configuration:
 
 - Imperative object configuration behavior is simpler and easier to understand.
-- As of Kubernetes version 1.5, imperative object configuration is more mature.
+- As of Kubernetes version 1.5, imperative object configuration was highlighted for its maturity. However, in the present Kubernetes landscape, both imperative (kubectl create) and declarative (kubectl apply) object configurations are well-established and widely used in production environments.
```
Comment (divya-mohan0209) [inline content/en/docs/concepts/overview/working-with-objects/object-management.md:116]: This is a section highlighting the advantages of the imperative object configuration. Therefore, IMO, even though the rephrasing is accurate, it doesn't fit this particular section.

Comment (haripriya9647) [reply content/en/docs/concepts/overview/working-with-objects/object-management.md:116]: Okay should the point be related to this "As of Kubernetes version 1.5, imperative object configuration is more mature." or can I add any other advantage

Comment (sftim) [reply content/en/docs/concepts/overview/working-with-objects/object-management.md:116]:
```suggestion
```

I'd drop the line.

Comment (haripriya9647) [reply content/en/docs/concepts/overview/working-with-objects/object-management.md:116]: Okay

Comment (sftim) [reply content/en/docs/concepts/overview/working-with-objects/object-management.md:116]:
```suggestion
```

Comment (sftim) [reply content/en/docs/concepts/overview/working-with-objects/object-management.md:116]: Feel free to make that change.

**Pull request closed without merging**